report_wkhtmltopdf_param/tests/test_report_paperformat.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017 Avoin.Systems
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+import odoo.tests
+from odoo.exceptions import ValidationError
+from odoo.tests.common import tagged
+
+
+@tagged("post_install", "-at_install")
+class TestWkhtmltopdf(odoo.tests.TransactionCase):
+    def test_wkhtmltopdf_incorrect_parameter(self):
+        for report_paperformat in self.env["report.paperformat"].search([]):
+            with (
+                self.assertRaises(ValidationError),
+                self.assertLogs(level=logging.WARNING),
```

Comment:
btw, if possible we should handle this `WARNING` instead of ignoring it

Reply:
`WARNING v18c_report_wkhtmltopdf_param odoo.addons.base.models.ir_actions_report: wkhtmltopdf: Exit with code 1 due to network error: HostNotFoundError`

This warning due to `wkhtmltopdf` is trying to resolve "bad-parameter" as an host. Recommend to catch warning log in test-case (v17 has same warning log)